Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -253,6 +253,7 @@ class FormulaAuditor
     def initialize(formula, options = {})
       @formula = formula
       @new_formula = options[:new_formula] && !formula.versioned_formula?
+      @versioned_formula = formula.versioned_formula?
```

Comment:
Maybe worth putting this on the line above and using `@versioned_formula` there?

Reply:
Fixed.